backend/tests/test_author_api.py

```diff
@@ -114,7 +116,9 @@ def test_update_password(self, client, test_user, friend_user):
         response = client.put("/author/update/", data=post_body_1,
                               content_type='application/json', charset='UTF-8')
         assert response.status_code == 200
-        assert User.objects.filter(password="liluzi").exists()
+        author = get_object_or_404(User, username=test_user.username)
```

Comment:
we can just do `author=request.user`, that will automatically resolve to an User object